lib/inMemFileDB.js

```diff
@@ -298,10 +298,12 @@ class InMemoryFileDB {
         }
 
         const clients = this.getClients();
+        let publishCount = 0;
 
         for (const i in clients) {
```

Comment:
What data type is `clients` supposed to be? Array? Object? Something else?

Reply:
Object

Reply:
added /** @returns {object} */

Reply:
`@returns` is only for functions. I asked because you are using the variable like an object, but the stub implementation returns an empty array instead.

Reply:
changed stub